frontend/package.json

```diff
@@ -18,7 +18,6 @@
     "@trussworks/react-uswds": "5.1.1",
     "@types/google-libphonenumber": "^7.4.23",
     "@types/history": "^4.7.10",
-    "@types/react-router-dom": "5.3.3",
```

Comment:
This type package was made obsolete by the upgrade to v6

https://stackoverflow.com/questions/72975579/do-i-still-need-this-types-react-router-dom-package-when-using-v6-of-react-rout